test/README.md

```diff
@@ -0,0 +1,13 @@
+# Test structure
+
+## Integration tests
+
+In this folder (`/test`) you should only put integration tests that care equally about several modules, or tests that don't relate to any single module.
```

Comment:
This folder could also house test helpers if we see a need for any.

Reply:
True, we already have the test-timer.js which I located with the tests. Should that one be here?